### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by Luligabi1**

### CODE REVIEW

File: br.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+			"featured_versions": "Versões destacadas",
+			"project_members": "Membros do projeto",
+			"technical_information": "Informações técnicas"
+		}
+	},
+	"status": {
+		"approved": "Aprovado",
+		"rejected": "Rejeitado",
+		"draft": "Rascunho",
+		"unlisted": "Deslistado",
```
Comment (flowln) [inline br.json:126]: Maybe `Oculto` would be a better translation? Or maybe `Não listado`?

**Review by flowln — COMMENTED** [review]
This should probably be `pt-BR.json`, shouldn't it? :slightly_smiling_face:

**Comment by Luligabi1** [comment]
> This should probably be `pt-BR.json`, shouldn't it? 🙂

maybe? US' english is simply `en` instead of `en-US` too.

**Comment by flowln** [comment]
> maybe? US' english is simply `en` instead of `en-US` too.

yeah, but I think it shouldn't be. There shouldn't be a "main" one and a "alternative" one. Like, pt-PT is just as valid as pt-BR, we shouldn't consider any one of those the "default portuguese" IMO

**Comment by Luligabi1** [comment]
> yeah, but I think it shouldn't be. There shouldn't be a "main" one and a "alternative" one. Like, pt-PT is just as valid as pt-BR, we shouldn't consider any one of those the "default portuguese" IMO
> 
> Also, `br`can be another language? idk

fair enough, I'll change it when I get home

File: br.json (hunk 1 above)
Comment (Luligabi1) [reply br.json]: Done

File: pt-BR.json [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+				"system": "Tema do sistema",
+				"light": "Modo claro",
+				"dark": "Modo escuro",
+				"oled": "Modo OLED"
+			},
+			"language": {
+				"label": "Selecionar linguagem",
+				"en": "English",
+				"en-GB": "English (UK)",
+				"br": "Português (Brasil)"
```
Comment (flowln) [inline pt-BR.json:29]:
```suggestion
				"pt-BR": "Português (Brasil)"
```

**Review by flowln — APPROVED** [review]
Seems like you forgot to change that little thing, otherwise looks great!

Obrigado! :)

**Comment by venashial** [comment]
Thank you! Portuguese is now on [Crowdin](https://crowdin.com/project/modrinth/pt-BR).

**Pull request closed without merging**